src/test/java/org/isf/medicalstockward/test/Tests.java

```diff
@@ -333,6 +338,7 @@ public void testIoNewMovementWard()
 			Double quantity = (double) medicalIoOperation.getCurrentQuantityInWard(wardTo, medical);
 			
 			_checkMovementWardIntoDb(movementWard.getCode());
+			jpa.commitTransaction();
```

Comment:
This is true, but if I run all tests, and not only this suite, I still get the same constraint violation, because of dirty context produced by other tests (although the in-memory DB).

See https://github.com/informatici/openhospital-core/pull/132#discussion_r456689646